java/semantickernel-core/src/main/java/com/microsoft/semantickernel/skilldefinition/DefaultSkillCollection.java

```diff
@@ -31,7 +31,7 @@ public ReadOnlySkillCollection build() {
         }
     }
 
-    protected Map<String, FunctionCollection> getSkillCollection() {
+    public Map<String, FunctionCollection> getSkillCollection() {
```

Comment:
I don't think we want to expose the skill collection and allow client to mutate the contents

In the C# code we have this `public IReadOnlySkillCollection ReadOnlySkillCollection { get; }`

Reply:
Inadvertent change. Backed out.